content/en/catalyst/fund9/challenges/grow-east-asia-grow-cardano/campaign-brief.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Grow East Asia, Grow Cardano"
+linkTitle: "Grow East Asia, Grow Cardano"
+type: docs
+hide_feedback: true
+description: "How do we grow the Cardano ecosystem in East Asia in an inclusive and supportive way? The same way we do in Africa, focused initiatives!"
+---
+<img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/12-Grow-East-Asia-Grow-Cardano-c2a227.png" style="width:500px;height500px">
+
+Fund Size: **250,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/399482) and [browse ideas]().
```

Comment:
Do we need to change the campaign link to this link, 'https://cardano.ideascale.com/c/campaigns/26602/about'?